go-controller/pkg/util/sriovnet_linux.go

```diff
@@ -0,0 +1,55 @@
+// +build linux
+
+package util
+
+import (
+	"github.com/Mellanox/sriovnet"
+)
+
+type SriovnetOps interface {
```

Comment:
naive question, if it seems we are using all the `"github.com/Mellanox/sriovnet"` directly, what is the benefit of mocking and having it here?

Reply:
Thats intended to be used in unit tests later, as in a unit-test it is undesirable to actually call sriovnet lib but provide a mock instead where calls may be checked.